toolboxes/core/gpu/cuNDArray_blas.h

```diff
@@ -20,31 +20,31 @@
 
 namespace Gadgetron{
 
-  template<class T> T dot( cuNDArray<T> *x, cuNDArray<T> *y, bool cc = true );
+  template<class T> T dot( cuNDArray<T> *x, cuNDArray<T> *y, size_t batchSize = INT_MAX, bool cc = true );
```

Comment:
I am unable to figure out what the `cc` argument does. As best as I can see it is not used? It caught my eye because you inserted the `batchSize` argument before the cc argument. Should we remove that?

Reply:
I would keep it around because it allows user to control if dot product should be x'.y  or x.y. ' - complex conjugate.